nixos/modules/services/networking/networkmanager.nix

```diff
@@ -244,6 +245,15 @@ in {
         '';
       };
 
+      firewallBackend = mkOption {
+        type = types.enum [ "iptables" "nftables" "none" ];
+        default = "iptables";
```

Comment:
And just now I'm wondering what the default prio for module options are. Like if they're `mkDefault` or `mkOverride` with a lower priority.

Reply:
If I understand `lib/modules.nix` correctly, `mkOptionDefault` is `1500` and `mkDefault` is `1000`, and the comments indicate that their current use is as intended. So I think that part is sorted now.